index.html

```diff
@@ -117,5 +123,44 @@
             "logo": "https://zenplayer.audio/img/zen-audio-player-905.png"
         }
     </script>
+	<!-- Load SVG defs for Plyr-->
```

Comment:
This loads SVG's for control's icons. They (Plyr github) suggest to use something like gulp-svg-sprite instead of this.